graph/GuidelinesGraph.md

```diff
@@ -328,6 +329,19 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+## External standards
+
+For ease of client use and interoperatibility, some APIs should implement a standard that is defined external to Microsoft Graph and OData. 
```

Comment:
From Mike, I should be clear what APIs "conflict with odata stnadrad/graph guidelines"